contracts/scripts/start.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+set -meuo pipefail
+
+function cleanup {
+  jobs -p | xargs kill
+}
+
+trap cleanup EXIT
+
+docker pull ethereum/client-go:stable
+
+CONTAINER=geth$RANDOM
+
+docker run --name $CONTAINER --rm -p8545:8545 ethereum/client-go:stable \
+  --http \
+  --http.api eth,web3,personal,net \
+  --http.addr=0.0.0.0 \
+  --http.vhosts='*' \
+  --dev \
+  --dev.period=0 \
+  &
```

Comment:
You could use our [docker-compose](https://github.com/web3well/bls-wallet/blob/main/docker-compose.yml#L20-L35) file to store this info instead.

```suggestion
container=geth${RANDOM} # non-exported bash vars should be lowercase

docker-compose up -d geth
docker rename geth ${container}
```

Reply:
The one in our docker-compose file is a bit different. Most notably, it binds the data directory to persist the state.

This version is ephemeral so it works the same way as `yarn hardhat node`, ie, you get a blank slate each time.

Would you want to change the one in docker-compose to fix this? We also need to change `dev.period` which is currently `14` in docker-compose.